examples/spi-demo/README.md

```diff
@@ -0,0 +1,14 @@
+# spi-demo
+SPIM demonstation code.
```

Comment:
When failing some LED is still turned off

Reply:
How you wrote it, the success and the all-fail test case would both result in 4 LEDs turned on.

> If all tests Led1 to 4 will light up, in case of error only the failing test
case will light up.

I am not sure if I understand you wrong or if that's really the case :)